OKTA-388434: Adds docs about the "co" operator and filters in the `/logs` API (#2101)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

| `actor.id eq ":id"`                          | Events that are published with a specific actor ID                                      |

> **Note:** SCIM filter expressions can't use the `published` attribute since it may conflict with the logic of the `since`, `after`, and `until` query params.
> Furthermore, using the `co` (contains) operator with the `debugContext.debugData.url` and `debugContext.debugData.requestUri` attributes is not supported and will

> > result in a 400 API response.

Reviewer: Maybe we don't need to specify the error response if the expression is not supported?

Author: That's actually the reason why we do show it.  If you look at the examples in this section, they follow the same pattern. @dandennhardt-okta since this was your suggestion, care to comment?

Reviewer: sounds good, i'll add the note back in

> > result in a 400 API response.

Reviewer: ```suggestion
> A request with an invalid SCIM filter expression returns an HTTP 400 API response.
```

```json
{
  "errorCode": "E0000053",

>   "errorSummary": "Invalid filter 'display_message eq \"Create okta user\"': Unrecognized attribute operator 'eqq' at position 16. Expected: eq,co,sw,pr,gt,ge,lt,le",

Reviewer: Are you sure you want to replace `eqq` with `eq`? Looks like that is the error here.

Author: `eqq` is not a valid operator.

Author: Oh totally missed that, you're right! Will revert.

}
```

> An invalid SCIM field / operator combination contained in a `filter` request parameter (e.g. `debugContext.debugData.url co "/oauth/"`) returns an HTTP 400 error with a message that indicates the unsupported combination, for example:

Reviewer: ```suggestion
An invalid SCIM field and operator combination within a `filter` request parameter (e.g. `debugContext.debugData.url co "/oauth/"`) returns an HTTP 400 error with a message that indicates the unsupported combination, for example:
```
